packages/flame/lib/src/camera/camera_component.dart

```diff
@@ -45,6 +46,29 @@ class CameraComponent extends Component {
   })  : viewport = viewport ?? MaxViewport(),
         viewfinder = viewfinder ?? Viewfinder();
 
+  /// Create a camera that shows a portion of the game world of fixed size
+  /// [width] x [height].
+  ///
+  /// The viewport will be centered within the canvas, expanding as much as
+  /// possible on all sides while maintaining the [width]:[height] aspect ratio.
+  /// The zoom level will be set such in such a way that exactly [width] x
+  /// [height] pixels are visible within the viewport. The viewfinder will be
+  /// initially set up to show world coordinates (0, 0) at the center of the
+  /// viewport.
+  factory CameraComponent.withFixedResolution({
+    required World world,
+    required double width,
+    required double height,
+    List<Component>? hudComponents,
```

Comment:
This should probably be added to the normal constructor too if we have it on this constructor.